devicemodel/samples/apl-mrb/sos_bootargs_release.txt

```diff
@@ -1 +1 @@
-console=tty0 i915.nuclear_pageflip=1 root=/dev/mmcblk1p1 rw rootwait quiet loglevel=3 no_timer_check consoleblank=0 i915.enable_initial_modeset=1 i915.tsd_init=7 i915.tsd_delay=2000 video=DP-1:d video=DP-2:d i915.avail_planes_per_pipe=0x01010F i915.domain_plane_owners=0x011111110000 i915.enable_guc_loading=0 i915.enable_guc_submission=0 i915.enable_preemption=1 i915.context_priority_mode=2 i915.enable_gvt=1 i915.enable_guc=0 cma=32M@0-
+console=tty0 i915.nuclear_pageflip=1 root=/dev/mmcblk1p1 rw rootwait quiet loglevel=3 no_timer_check consoleblank=0 i915.enable_initial_modeset=1 i915.tsd_init=7 i915.tsd_delay=2000 video=DP-1:d video=DP-2:d i915.avail_planes_per_pipe=0x01010F i915.domain_plane_owners=0x011111110000 i915.enable_guc_loading=0 i915.enable_guc_submission=0 i915.enable_preemption=1 i915.context_priority_mode=2 i915.enable_gvt=1 i915.enable_guc=0 cma=64M@0-
```

Comment:
Do we update the doc too?

Reply:
Unless I've missed it, that parameter (and its value) is not referenced in any of our doc so we should be good here.